package.json

```diff
@@ -4,16 +4,17 @@
   "description": "A react UI library for resmio",
   "main": "dist/index.js",
   "scripts": {
-    "styleguide-server": "styleguidist server",
-    "styleguide-build": "styleguidist build",
     "build": "webpack",
     "build:production" : "npm run build --production",
     "lint": "eslint . --ext .js --ext .jsx || true",
     "lint:fix": "eslint . --ext .js --ext .jsx --fix || true",
-    "prepublish": "npm run build:production && npm run styleguide-build --production",
+    "prepublish": "npm run build:production",
+    "preversion": "npm run styleguide-build --production",
```

Comment:
because prepublish is run when you do `npm install` for some reason - I just separated it out so we weren't building the style guide everytime